### DISCUSSION TITLE
Categorical value encoding

### DISCUSSION THREAD

**Discussion by matheger** [biolab/orange3#5243] [Q&A] [answered]
I've been looking around to easily encode categorical data into integer values such as with sklearn's `OneHotEncoder`, but I couldn't find anything straightforward. The closest thing I see is to use the "Edit Domain" widget and then manually assign numeric values in the "Values" field; but that seems awkward and tedious (and doesn't support one-hot encoding). 

Is this implemented anywhere else?

**Comment by janezd** [comment, marked as answer]
Does widget Continuize (admittedly not a great name) help here?

**Reply by matheger** [reply]
Ha! Indeed, that seems to do the trick. No idea why I missed it before. But yeah, "Continuize" isn't really a good name.

**Reply by irgolic** [reply]
What did you search for? We can add keywords to Continuize, so it shows up in the QuickMenu (search menu).

**Reply by matheger** [reply]
I didn't search, I'm a new user and was looking through the widgets one by one to see what was implemented. For the search, keywords like "one-hot", "encode", "dummy" etc. would probably be a good fit. Maybe take inspiration from the method names in the [sklearn preprocessing documentation](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.preprocessing).
